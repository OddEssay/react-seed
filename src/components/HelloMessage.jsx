import React from 'react'

export const HelloMessage = React.createClass({
  render() {
    return <div>Hello {this.props.params.name}</div>
  }
})
