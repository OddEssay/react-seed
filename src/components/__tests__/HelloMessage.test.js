import React from 'react'
import renderer from 'react-test-renderer'
import { HelloMessage } from '../HelloMessage'
test('HelloMessage', () => {
  const tree = renderer.create(
    <HelloMessage params={ {name: 'World'} } />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
