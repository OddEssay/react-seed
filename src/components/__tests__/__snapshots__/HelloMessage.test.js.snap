exports[`test HelloMessage 1`] = `
<div>
  Hello 
  World
</div>
`;
